server/test/fixtures/invalid_config.yml

```diff
@@ -0,0 +1,204 @@
+server:
+  app:
+    not_a_field: true
+    verbose: false
+    debug: false
+    host: localhost
+    port : null
+    open_browser: false
+    force_https: false
+    flask_secret_key: null
+    generate_cache_control_headers: false
+    server_timing_headers: false
+#    csp_directives: null  ## should throw an error
```

Comment:
It's super difficult to understand what in this config is wrong. Is it possible to simply have the error itself only and layer it on the default config to generate the error?

Reply:
Yeah, it is not an error have missing config parameters.  A user's config file need only have the parameters they want to change from the defaults.

Reply:
removed this file for clarity, instead I'll just generate an invalid config by passing in a var of the wrong type